dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -6,7 +6,7 @@
 
 import pandas as pd
 import numpy as np
-
+import json
```

Comment:
json is an first-party package, it should be grouped with the imports at the top of the file
```python
# first party imports

# third party

# our lib
```